README.md

```diff
@@ -1,20 +1,157 @@
+
+After loading the visualizations via the following API call, you will see the following steps: TBC
+
+
+
+
+
+
+
+
+# <a name="mitigation"></a>
+# Unfairness Mitigation Techniques
+
+
+
+
+
+Fairlearn includes a Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+## Pre-processing and In-processing:
+
+Fairlearn shows that given access to a learning oracle for a class H, there is an efficient algorithm to find the lowest-error distribution over classifiers in H subject to equalizing false positive rates across polynomially many subgroups.
 
 | algorithm | description | classification/regression | protected attributes | supported fairness definitions |
 | --- | --- | --- | --- | --- |
-| `fairlearn.` `expgrad` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
-| `fairlearn.` `reductions.` `GridSearch` | As described in Section 3.4 in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP |
-| `fairlearn.` `reductions.` `GridSearch` | Grid Search for regression | regression | binary | BGL |
-| `fairlearn.` `post_processing.` `ROCCurveBasedPostProcessing` | Post-processing algorithm based on the paper [Equality of Opportunity in Supervised Learning](https://arxiv.org/pdf/1610.02413.pdf)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `ExponentiatedGradient` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `GridSearch` | Black-box approach described in Section 3.4 of the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP |
+| `fairlearn.` `reductions.` `GridSearch` | Grid search for regression | regression | binary | BGL |
+
+
+
+
+## Post-processing: 
+| algorithm | description | classification/regression | protected attributes | supported fairness definitions |
+| --- | --- | --- | --- | --- |
+| `fairlearn.` `post_processing.` `ThresholdOptimizer` | Post-processing algorithm based on the paper [Equality of Opportunity in Supervised Learning](https://arxiv.org/pdf/1610.02413.pdf). This technique takes as input any classifier’s prediction and the sensitive attribute, and derives a monotone transformation of the classifier’s prediction to remove any disparity with respect to the training examples. This postprocessing technique developes a method for constructing randomized decision rules that ensure equal opportunity and equalized odds are equal across protected groups.
+
+| binary classification | categorical | DP, EO |
```

Comment:
will this work if it's in a new line? Shouldn't it be part of the same line that the ThresholdOptimizer description is? Does it render properly for you?